lib/dfu.js

```diff
@@ -108,12 +109,16 @@ var that = module.exports = {
 		];
 
 		that.checkBinaryAlignment("-D " + binaryPath);
-		return utilities.deferredSpawnProcess("dfu-util", args);
+		return utilities.deferredChildProcess(that.getCommand() + ' ' + args.join(' '));
 	},
 
-	getCommandPrefix: function () {
-		var sudo = (settings.useSudoForDfu) ? "sudo " : "";
-		return sudo + "dfu-util -d " + that.deviceID;
+	getCommand: function () {
```

Comment:
Just to clarify on this name: I decided to use `getCommand()` because this method is located in dfu module ( and actually permit to get the proper dfu command to run ).